packages/mgt-element/src/components/baseComponent.ts

```diff
@@ -291,6 +291,7 @@ export abstract class MgtBaseComponent extends LitElement {
 
   private async handleActiveAccountUpdates() {
     this.clearState();
+    await Providers.getCacheId();
```

Comment:
This call lives in BaseComponent, so every component instance on the page will each attempt to initialize the cache id. Alternatively, this could be done lazily; Meaning you don't actually need to initialize the cache id explicitly, because it will initialize itself on first use.

Reply:
The problem I was facing was that the getcache calls from the components were getting called before the cache ID was actually set, that's why I called it here before any component requests data from the cache. And I thought it would be okay because the cacheID is stored in memory so if it already exists then there won't be an extra API call.

Reply:
Do you have a suggestion for where I can put this call instead?